src/js/pages/takes/newComponents/TakeCard/TakeCard.js

```diff
@@ -143,11 +154,11 @@ export class TakeCard extends React.Component {
               audioFile={config.streamingUrl+this.props.location}  playAudio={this.props.play}
               playing = {this.state.takePlaying} durationTime={this.props.duration}
               pos = {this.state.pos}
-              options= {{ cursorWidth: 2, progressColor: '#009CFF', cursorColor: '#E74C3C', barWidth: 1, hideScrollbar: true, normalize: true, height: 35, waveColor: '#969595' }}
+              options= {{ cursorWidth: 2, progressColor: '#009CFF', cursorColor: '#E74C3C', barWidth: 3.5, hideScrollbar: true, normalize: true, height: 65, waveColor: '#969595' }}
```

Comment:
i think just decrease the bar width a little bit. it looks a little to chunky and looks a little bigger than the mock ups as well 
<img width="275" alt="screen shot 2018-04-18 at 3 03 10 pm" src="https://user-images.githubusercontent.com/25537741/38952485-db780eac-4319-11e8-8162-306cc5790756.png">
<img width="276" alt="screen shot 2018-04-18 at 3 03 16 pm" src="https://user-images.githubusercontent.com/25537741/38952486-db83afbe-4319-11e8-856b-5dd889836b8f.png">

Reply:
you are right!! my bad, I'll decrease the bar width